files-community/Files
# Fix: Fixed grip splitter cursor behavior for preview pane and secondary pane

**Resolved / Related Issues**
   Closes #12046

Also simplifies a bit the logic for changing the grid splitter cursor for DetailsLayoutBrowser and MainPage.

**Validation**
How did you test these changes?
   1. Check cursor behaviour when hovering over the pane splitter
   2. Check cursor behaviour when hovering over the preview pane splitter

**Screenshots (optional)**
Add screenshots here.
